pyproject.toml

```diff
@@ -170,13 +170,13 @@ filterwarnings = [
 addopts = "-ra"
 
 [tool.cibuildwheel]
-build = "cp311-* cp312-* cp313-*"
 build-frontend = "build[uv]; args: -Csetup-args='-Dpython.allow_limited_api=true'"
 build-verbosity = 1
 test-groups = ["test", "concurrency"]
+test-sources = "tests"
```

Comment:
Depending exactly how the final 3.0.0 behaves, this line may actually not serve any purpose for an `src` layout package.